src/runtimes/docker/clients/DockerClientBase/DockerInfoRecord.ts

```diff
@@ -5,17 +5,30 @@
 
 import { ContainerOS, isContainerOS } from "../../contracts/ContainerClient";
 
+export type DockerPlugins = {
+    Name?: string;
+};
+
+export type DockerClientInfo = {
+    Plugins?: DockerPlugins[];
+};
+
 export type DockerInfoRecord = {
     OperatingSystem?: string;
     OSType?: ContainerOS;
+    ClientInfo?: DockerClientInfo;
 };
-
 export function isDockerInfoRecord(maybeInfo: unknown): maybeInfo is DockerInfoRecord {
+
     const info = maybeInfo as DockerInfoRecord;
 
     if (typeof info.OSType === 'string' && !isContainerOS(info.OSType)) {
         return false;
     }
-
-    return true;
+    else if (!info.ClientInfo) {
+        return false;
+    }
+    else {

```

Comment:
The return true shouldn't need to be wrapped in an else statement here.

Reply:
I agree